core/include/kernel/boot.h

```diff
@@ -105,4 +105,23 @@ void discover_nsec_memory(void);
 /* Add reserved memory for static shared memory in the device-tree */
 int mark_static_shm_as_reserved(struct dt_descriptor *dt);
 
+#ifdef CFG_BOOT_MEM
```

Comment:
A bit of description could help. Maybe something like what's in the commit message:
```
Stack-like memory allocations during boot before a heap has
been configured. Boot_mem heap considers possible relocation
during virtual memory initialization. Unused memory is unmapped
and released to SECRAM_OVERALL allocation pool once MMU is
initialized.
```
(if I understood correctly)

Reply:
Thanks, I'll add something like that.